doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -133,17 +135,26 @@ size isn't smaller than 4% of ``block``, because RGW heavily uses it to store it
 metadata. For example, if the ``block`` size is 1TB, then ``block.db`` shouldn't
 be less than 40GB. For RBD workloads, 1% to 2% of ``block`` size is usually enough.
 
-If *not* using a mix of fast and slow devices, it isn't required to create
-separate logical volumes for ``block.db`` (or ``block.wal``). Bluestore will
-automatically manage these within the space of ``block``.
+In releases prior to Pacific, DB internal level sizes meant that the DB could
```

Comment:
@neha-ojha Does this description of DB sizing, and the changes that I understand will come with Pacific, make sense to you?

Reply:
@ifed01 @aclamk how does this sound to you?

Reply:
I've seen some commits toward sharding RocksDB or otherwise improving the stranding of DB device capacity that doesn't align with the levels, but am unsure exactly what this will mean in Pacific, so I want to be accurate here.  The percent-based sizing that's been recommended for a couple of years has misled a lot of people.  With RBD clusters few can afford the 4% that is recommended, and say someone provisions 20GB, they're confused when only like 3GB gets used and the rest spills to the slow device.

More than once I've been tempted to take out the percentage-based sizing recommendations, since with existing releases they don't match reality.  If we shard or otherwise are able to better use arbitrarily sized DB devices in Pacific, then we'd want to leave it in.

I've also debated documenting the use of `max_bytes_for_level_base` to adjust RocksDB to whatever capacity is available, as @ksingh7 describes in his billion-objects presentation, but without knowledge of the ramifications of doing so I'm hesitant to encourage people to shoot themselves in the foot.  @markhpc has posted about this too.